Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -110,6 +110,7 @@ void Window::visit_edges(JS::Cell::Visitor& visitor)
         visitor.visit(plugin_object);
     for (auto& mime_type_object : m_pdf_viewer_mime_type_objects)
         visitor.visit(mime_type_object);
+    visitor.visit(m_count_queuing_strategy_size_function);
```

Comment:
Missing a visit to the byte length strategy

Reply:
whoops!